lib/packages/libbsdgames2.rb

```diff
@@ -0,0 +1,85 @@
+require 'package'
```

Comment:
We need to create a `Libbsdgames2` class otherwise user will get errors when `crew install libbsdgames2`

Reply:
libbsdgames2 isn't supposed to be installed. It's supposed to provide centralized information for a bunch of other package files.

Reply:
Make a message for telling user about that?

Reply:
Actually it looks like it's not even an issue. It'll just say that the package isn't found.
```
$ crew install libbsdgames2
Package libbsdgames2 not found. :(
$ crew install bsdgame
Package bsdgame not found. :(
```